app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,11 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] As requested by the previous reviewer, the comment controller, should equally have the new action. It should look like the piece of code below:

```ruby
  def new
    @comment = Comment.new
  end
```